README.md

```diff
@@ -30,8 +30,11 @@ For Arch Linux, there is a package available in the [AUR](https://aur.archlinux.
    * Stores your files in a folder/prefix called `git-annex`:
 
 ```
-git annex initremote google type=external externaltype=googledrive prefix=git-annex chunk=50MiB encryption=shared mac=HMACSHA512
+git annex initremote google type=external externaltype=googledrive prefix=git-annex root_id=<some_id> chunk=50MiB encryption=shared mac=HMACSHA512
```

Comment:
This is supposed to be the default example users can just copy & paste and maybe change the prefix. As the shared folder scenario is more of a special case, I'd rather put it into an extra list of optional parameters.